packages/theming/src/utils/getColor.ts

```diff
@@ -89,21 +86,11 @@ const toColor = (
 
   if (typeof _hue === 'object') {
     retVal = toHex(_hue, shade, offset, scheme);
-  } else if (_hue === 'transparent' || valid(_hue)) {
+  } else if (_hue === 'transparent' || isValidColor(_hue)) {
     if (shade === undefined) {
       retVal = _hue;
     } else {
-      const _colors = scale([PALETTE.white, _hue, PALETTE.black])
-        .correctLightness()
-        .colors(PALETTE_SIZE + 2); // add 2 to account for the white and black endpoints removed below
-
-      _hue = _colors.reduce<Record<number, string>>((_retVal, color, index) => {
-        if (index > 0 && index <= PALETTE_SIZE) {
-          _retVal[index * 100] = color;
-        }
-
-        return _retVal;
-      }, {});
```

Comment:
Can you add this style of hue generation to the sandbox for comparison? If it isn't a compelling difference, I'd prefer to retain this for being more straightforward and imperative with respect to Garden's palette size. Garden design will be able to help us evaluate the trade-offs.

Reply:
Added to the [Sandbox](https://stackblitz.com/edit/stackblitz-starters-s2jbrj?file=src%2FApp.tsx) under `Chroma 12 Current`.

Reply:
Awesome. Thanks so much.

Reply:
@ze-flo did you look at https://color2k.com/#get-scale or rule it out?

Reply:
Excellent suggestion! 💯 🔥 🚀  

Using a binary search and a 100 color scale, `color2k` outputs excellent results despite its tiny footprint. I think we have a winner! 🍾 🎉 🥳 

![Screen Shot 2024-04-30 at 3 11 18 PM](https://github.com/zendeskgarden/react-components/assets/6879688/ba70ef36-864f-43d2-9c03-491186d00f50)

![Screen Shot 2024-04-30 at 3 15 25 PM](https://github.com/zendeskgarden/react-components/assets/6879688/498ffeb9-6ec2-4a21-9d59-80bf9b7cc55b)

Check out the new and improved [demo](https://stackblitz.com/edit/vitejs-vite-1v4hhp?file=src%2FApp.tsx).